bids/layout/models.py

```diff
@@ -491,6 +491,10 @@ def __init__(self, file, entity, value, dtype=None):
         self._dtype = dtype
 
         self._init_on_load()
+    
+    def __repr__(self):
+        msg = "Tag for file '{}' with entity name '{}' has value {}."
```

Comment:
I'd prefer to keep the `repr` looking like an obvious object repr, with `<>` and all. Maybe something like:

```suggestion
        msg = "<Tag file:{!r} entity:{!r} value:{!r}>"
```